### DISCUSSION TITLE
summary must be and in Command

### DISCUSSION THREAD

**Discussion by Andrey-2021** [CommunityToolkit/dotnet#420] [Ideas]
When I write summary about Method,
this summary automatically must be and in Command.
So that I can see it (summary) in Command Descriptions/ IntelliSense

`/// <summary>
/// MyInfo about method
/// </summary>
[RelayCommand]
public async void UpdateDatas()
.....
.....
UpdateDatasCommand.Execute(null);`

When I hover my mouse over a word <UpdateDatasCommand> VStudio must be show Summary

**Comment by michael-hawker** [comment]
@Sergio0694 I think the request here is to forward the method comment to that of the Command property that's generated? Can roslyn inspect the comments attached to methods as well? I would hope so?
